src/ir.cpp

```diff
@@ -757,7 +757,9 @@ Stmt CallStmt::make(std::vector<Var> results,
 
 // struct Scope
 Stmt Scope::make(Stmt scopedStmt) {
-  iassert(scopedStmt.defined());
+  if (!scopedStmt.defined()) {
```

Comment:
Yup, looks like this was a remnant of a previous iteration of a GPU lowering pass I was working on. Reintroduced the assert in #47